stestr/tests/test_return_codes.py

```diff
@@ -245,16 +243,16 @@ def test_combine_results(self):
         self.assertRunExit('stestr run passing', 0)
         stdout = self._get_cmd_stdout(
             'stestr last --no-subunit-trace')
-        stdout = six.text_type(stdout[0])
+        stdout = str(stdout[0])
         test_count_split = stdout.split(' ')
         test_count = test_count_split[1]
         test_count = int(test_count)
-        id_regex = re.compile('\(id=(.*?)\)')
+        id_regex = re.compile(r'\(id=(.*?)\)')
```

Comment:
I'm not sure we need these changes(adding `r`) now. But it should work correctly.
We still have a lot of `re.compile` in the rest of the codes, and we can update them later if we need it.

Reply:
Yeah, I just ran pyupgrade (which might not have been the best idea) which did this conversion to raw strings because we're escaping the `'('`. I'm not sure it hurts to do this, it should only be needed if we are escaping anything in the regex.

Reply:
ok, sure